### PR TITLE
24 5

### PR DESCRIPTION
**Pull request by victorputrovaspose**

### CODE REVIEW

File: slides/developer-guide/working-with-charts/working-with-categories/add-a-chart-category/_index.md
Comment (andreypotapovaspose) [inline slides/developer-guide/working-with-charts/working-with-categories/add-a-chart-category/_index.md:1]: The changes in this file contradict the pictures.

File: slides/developer-guide/working-with-charts/working-with-categories/delete-a-chart-category/_index.md
Comment (andreypotapovaspose) [inline slides/developer-guide/working-with-charts/working-with-categories/delete-a-chart-category/_index.md:1]: The changes in this file contradict the pictures.

File: slides/developer-guide/working-with-charts/working-with-categories/update-a-chart-category/_index.md
Comment (andreypotapovaspose) [inline slides/developer-guide/working-with-charts/working-with-categories/update-a-chart-category/_index.md:1]: The changes in this file contradict the pictures.

File: slides/developer-guide/working-with-charts/working-with-data-points/add-a-chart-data-point/_index.md
Comment (andreypotapovaspose) [inline slides/developer-guide/working-with-charts/working-with-data-points/add-a-chart-data-point/_index.md:1]: The changes in this file contradict the pictures.

File: slides/developer-guide/working-with-charts/working-with-data-points/delete-a-chart-data-point/_index.md
Comment (andreypotapovaspose) [inline slides/developer-guide/working-with-charts/working-with-data-points/delete-a-chart-data-point/_index.md:1]: The changes in this file contradict the pictures.

File: slides/developer-guide/working-with-charts/working-with-data-points/update-a-chart-data-point/_index.md
Comment (andreypotapovaspose) [inline slides/developer-guide/working-with-charts/working-with-data-points/update-a-chart-data-point/_index.md:1]: The changes in this file contradict the pictures.

File: slides/developer-guide/working-with-charts/working-with-series/add-a-chart-series/_index.md
Comment (andreypotapovaspose) [inline slides/developer-guide/working-with-charts/working-with-series/add-a-chart-series/_index.md:1]: The changes in this file contradict the pictures.

File: slides/developer-guide/working-with-charts/working-with-series/delete-a-chart-series/_index.md
Comment (andreypotapovaspose) [inline slides/developer-guide/working-with-charts/working-with-series/delete-a-chart-series/_index.md:1]: The changes in this file contradict the pictures.

File: slides/developer-guide/working-with-charts/working-with-series/update-a-chart-series/_index.md
Comment (andreypotapovaspose) [inline slides/developer-guide/working-with-charts/working-with-series/update-a-chart-series/_index.md:1]: The changes in this file contradict the pictures.

File: slides/developer-guide/working-with-charts/working-with-series/update-a-series-group/_index.md
Comment (andreypotapovaspose) [inline slides/developer-guide/working-with-charts/working-with-series/update-a-series-group/_index.md:1]: The changes in this file contradict the pictures.

File: slides/developer-guide/working-with-documents/create-new-presentations/use-a-pdf-document/_index.md [hunk 1]
```diff
@@ -31,23 +31,41 @@ The following method allows you to create a PowerPoint presentation from a PDF d
 | :- | :- | :- | :- | :- |
 |name|string|path|true|The file name of the new presentation or existing one.|
 |pdf|file|formData|true|The input data of a PDF document.|
+|options|[PdfImportOptions](#import-options)|header|false|The password to open the presentation.|
```
Comment (andreypotapovaspose) [inline slides/developer-guide/working-with-documents/create-new-presentations/use-a-pdf-document/_index.md:34]: There is an incorrect comment for the `options` parameter here.

File: slides/developer-guide/working-with-documents/create-new-presentations/use-a-pdf-document/_index.md [hunk 2]
```diff
@@ -205,7 +223,8 @@ class Test
 
         using (var pdfStream = File.OpenRead("test.pdf"))
         {
-            var response = api.ImportFromPdf("summary.pptx", pdfStream);
+            PdfImportOptions options = new PdfImportOptions();
+            var response = api.ImportFromPdf("summary.pptx", pdfStream, options);
```
Comment (andreypotapovaspose) [inline slides/developer-guide/working-with-documents/create-new-presentations/use-a-pdf-document/_index.md:227]: The `DetectTables` option has not been aplied here.

File: slides/developer-guide/working-with-files-folders-and-storage/working-with-folders/_index.md [hunk 3]
```diff
@@ -760,10 +760,10 @@ class Application
         SlidesApi slidesApi = new SlidesApi("MyClientId", "MyClientSecret");
 
         string storageName = "MyStorage";
-        string folderPath = "MyFolder/MyPresentations";
+        string folderToDelete = "MyFolder/MyPresentations";
```
Comment (andreypotapovaspose) [inline slides/developer-guide/working-with-files-folders-and-storage/working-with-folders/_index.md:763]: This is a very simple code example and in this case, such clarifications are unnecessary.

File: slides/developer-guide/working-with-shapes/working-with-shapes-on-special-slides/working-with-text-portions-on-a-special-slide/create-a-new-text-portion-on-a-special-slide/_index.md
Comment (andreypotapovaspose) [inline slides/developer-guide/working-with-shapes/working-with-shapes-on-special-slides/working-with-text-portions-on-a-special-slide/create-a-new-text-portion-on-a-special-slide/_index.md:1]: The changes in this file contradict the pictures.

File: slides/developer-guide/working-with-shapes/working-with-shapes-on-special-slides/working-with-text-portions-on-a-special-slide/delete-text-portions-on-a-special-slide/_index.md
Comment (andreypotapovaspose) [inline slides/developer-guide/working-with-shapes/working-with-shapes-on-special-slides/working-with-text-portions-on-a-special-slide/delete-text-portions-on-a-special-slide/_index.md:1]: The changes in this file contradict the pictures.

File: slides/developer-guide/working-with-shapes/working-with-shapes-on-special-slides/working-with-text-portions-on-a-special-slide/get-text-portion-properties-from-a-special-slide/_index.md
Comment (andreypotapovaspose) [inline slides/developer-guide/working-with-shapes/working-with-shapes-on-special-slides/working-with-text-portions-on-a-special-slide/get-text-portion-properties-from-a-special-slide/_index.md:1]: The changes in this file contradict the pictures.

File: slides/developer-guide/working-with-shapes/working-with-shapes-on-special-slides/working-with-text-portions-on-a-special-slide/update-text-portions-on-a-special-slide/_index.md
Comment (andreypotapovaspose) [inline slides/developer-guide/working-with-shapes/working-with-shapes-on-special-slides/working-with-text-portions-on-a-special-slide/update-text-portions-on-a-special-slide/_index.md:1]: The changes in this file contradict the pictures.

**Review by andreypotapovaspose — CHANGES_REQUESTED** [review]

File: slides/developer-guide/working-with-charts/working-with-categories/add-a-chart-category/_index.md
Comment (victorputrovaspose) [reply slides/developer-guide/working-with-charts/working-with-categories/add-a-chart-category/_index.md:1]: let us remove the contradictions after using new testdata in tests

File: slides/developer-guide/working-with-documents/create-new-presentations/use-a-pdf-document/_index.md (hunk 1 above)
Comment (victorputrovaspose) [reply slides/developer-guide/working-with-documents/create-new-presentations/use-a-pdf-document/_index.md]: will be fixed

File: slides/developer-guide/working-with-documents/create-new-presentations/use-a-pdf-document/_index.md (hunk 2 above)
Comment (victorputrovaspose) [reply slides/developer-guide/working-with-documents/create-new-presentations/use-a-pdf-document/_index.md]: will be fixed

**Review by andreypotapovaspose — APPROVED** [review]

**Pull request merged**